scripts/publish.ts

```diff
@@ -18,81 +22,89 @@ function branchToTag(branch: string): string {
   return 'integration'
 }
 
-function isPatchBranch(version: string): boolean {
-  return /^2\.(\d+)\.x/.test(version)
+/** Is this a prisma-engines patch branch? (ends in .x) */
+function isPatchBranch(branchName: string): boolean {
+  return /^(\d+)\.(\d+)\.x/.test(branchName)
```

Comment:
This might be dangerous, to go from `2` to `(\d+)` but is prep for a major release.
Please double check the regex.

Reply:
looks good